### PR TITLE
[root]: Butifying the link list

### LINKED ISSUE #3044: [root]: Add a single intro link instead of links to chapter in the root of the Anuket specs

### PR DESCRIPTION
**Pull request by CsatariGergely**
Closes #3044

### CODE REVIEW

**Comment by collivier** [comment]
Please rebase it (I cannot do it in Nokia's repo) - now gate is successful

**Comment by CsatariGergely** [comment]
> Please rebase it (I cannot do it in Nokia's repo) - now gate is successful

@collivier  Done now.

File: doc/index.rst [hunk 1]
```diff
@@ -1,19 +1,28 @@
 Anuket Specifications
 =====================
 
+-  :ref:`Introduction <common/README:anuket project>`
+-  :doc:`Reference Model (RM) <ref_model:index>`
+-  :doc:`OpenStack based Reference Architecture (RA1) <ref_arch_openstack:index>`
+-  :doc:`Kubernetes based Reference Architecture (RA2) <ref_arch_kubernetes:index>`
+-  :doc:`Reference Conformance for OpenStack (RC1) <ref_cert_RC1:index>`
+-  :doc:`Reference Conformance for Kubernertes (RC2) <ref_cert_RC2:index>`
+-  :doc:`Reference Implementation based on OpenStack (RI1) <ref_impl_cntt-ri:index>`
```
Comment (pgoyal01) [inline doc/index.rst:10]: should we use this change to make the titles for RI1 and RI2 correct:
```suggestion
-  :doc:`Reference Implementation based on RA1 specifications (RI1) <ref_impl_cntt-ri:index>`
```

File: doc/index.rst [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
+-  :doc:`Reference Implementation based on Kubernetes (RI2) <ref_impl_cntt-ri2:index>`
```
Comment (pgoyal01) [inline doc/index.rst:11]:
```suggestion
-  :doc:`Reference Implementation based on RA2 specifications (RI2) <ref_impl_cntt-ri2:index>`
```

File: doc/index.rst [hunk 3]
```diff
@@ -1,19 +1,28 @@
 Anuket Specifications
 =====================
 
+-  :ref:`Introduction <common/README:anuket project>`
+-  :doc:`Reference Model (RM) <ref_model:index>`
+-  :doc:`OpenStack based Reference Architecture (RA1) <ref_arch_openstack:index>`
+-  :doc:`Kubernetes based Reference Architecture (RA2) <ref_arch_kubernetes:index>`
+-  :doc:`Reference Conformance for OpenStack (RC1) <ref_cert_RC1:index>`
```
Comment (pgoyal01) [inline doc/index.rst:8]:
```suggestion
-  :doc:`Reference Conformance (RC1) for RA1 based Implementations <ref_cert_RC1:index>`
```

File: doc/index.rst [hunk 4]
```diff
@@ -1,19 +1,28 @@
...
+-  :doc:`Reference Conformance for Kubernertes (RC2) <ref_cert_RC2:index>`
```
Comment (pgoyal01) [inline doc/index.rst:9]:
```suggestion
-  :doc:`Reference Conformance (RC2) for RA2 based Implementations <ref_cert_RC2:index>`
```

File: doc/index.rst [hunk 5]
```diff
@@ -1,19 +1,28 @@
 Anuket Specifications
 =====================
 
+-  :ref:`Introduction <common/README:anuket project>`
+-  :doc:`Reference Model (RM) <ref_model:index>`
+-  :doc:`OpenStack based Reference Architecture (RA1) <ref_arch_openstack:index>`
```
Comment (pgoyal01) [inline doc/index.rst:6]:
```suggestion
-  :doc:`Reference Architecture (RA1) for OpenStack based cloud infrastructure <ref_arch_openstack:index>`
```

File: doc/index.rst [hunk 6]
```diff
@@ -1,19 +1,28 @@
...
+-  :doc:`Kubernetes based Reference Architecture (RA2) <ref_arch_kubernetes:index>`
```
Comment (pgoyal01) [inline doc/index.rst:7]:
```suggestion
-  :doc:`Reference Architecture (RA2) for Kubernetes based cloud infrastructure <ref_arch_kubernetes:index>`
```

**Review by ulikleber — APPROVED** [review]

**Review by ildikov — APPROVED** [review]

**Comment by gkunz** [comment]
sorry for the very late comment... If I understand the old solution correctly, Sphinx uses the heading of the index file pointed to for the name of the link. So instead of overwriting the links explicitly here, wouldn't a more consistent approach be to ensure that the headings in the target index files correctly named? Then we'd have the same name in both places instead of overwriting it here and once clicking on it, the heading is different. This can be confusing.

**Comment by CsatariGergely** [comment]
Yes I agree, that it would be more consistent, but that looks like a multi release project :)
What if we merge this as it is and I create the PR-s to update the target document titles and we gradually remove the overwites from here as the correction PR-s are merged?

**Comment by gkunz** [comment]
@CsatariGergely: that's totally fine with me. Let's separate the concept from the implementation. If we agree to aim for consistency, we can surely gradually update the docs to get there.

**Review by gkunz — APPROVED** [review]

**Review by xudan2189 — APPROVED** [review]

**Review by rgstori — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

**Pull request merged**